component/distributed_cache/distributed_cache.go

```diff
@@ -71,7 +71,7 @@ type DistributedCache struct {
 	safeDeletes         bool
 	cacheAccess         string
 
-	azstroage internal.Component
+	azstorage internal.Component
 }
```

Comment:
likewise, changes to this file can be kept in the other PR which we have for the distributed_cache package.